File: Javascript/ejercicio16.js
//Algoritmo DiaDeLaSemana
// Definir numero Como Entero
let numero;
do {
	numero = parseInt(prompt("Ingrese un número del 1 al 7:").trim());

	if (numero === 1) {
		console.log("Lunes");
		alert("Lunes");
	} else if (numero === 2) {
		console.log("Martes");
		alert("Martes");
	} else if (numero === 3) {
		console.log("Miercoles");
		alert("Miercoles");
	} else if (numero === 4) {
		console.log("Jueves");
		alert("Jueves");
	} else if (numero === 5) {
		console.log("Viernes");
		alert("Viernes");
	} else if (numero === 6) {
		console.log("Sabado");
		alert("Sabado");
	} else if (numero === 7) {
		console.log("Domingo");
		alert("Domingo");
	} else {
		console.log("Número inválido. Intente nuevamente.");
		alert("Número inválido. Intente nuevamente.");
	}
} while (numero >= 1 || numero <= 7);

File: Phython/practica/ejercicio16.py
# Algoritmo DiaDeLaSemana
#Definir numero Como Entero

while True:
    numero = int(input("Ingrese un número del 1 al 7: "))
    if 1 <= numero <= 7:
        break
    else:
        print("Número inválido. Intente nuevamente.")

if numero == 1:
	print("Lunes")
		
elif numero == 2:
	print("Martes")
      
elif numero == 3:
	print("Miercoles")

elif numero == 4:
	print("Jueves")

elif numero == 5:
	print("Viernes")
	
elif numero == 2:
	print("Sabado")
	
elif numero == 2:
	print("Domingo")
	
else:
	print("Número inválido. Intente nuevamente.")
	